app/Http/Controllers/Legacy/Web/CampaignsController.php

```diff
@@ -18,56 +19,74 @@ public function __construct()
         $this->middleware('role:admin,staff', ['only' => ['store', 'update', 'destroy']]);
     }
 
+    /**
+     * Show overview of campaigns and their IDs.
+     */
+    public function index()
+    {
+        $campaigns = Campaign::all();
```

Comment:
We probably want to [paginate](https://laravel.com/docs/5.7/pagination) this index page.

Reply:
ah great call @DFurnes ! Updated this to have pagination (25 per page for now). Here's what it looks like (limited to 5 so you could see the pagination). 

![screen shot 2018-11-06 at 5 09 27 pm](https://user-images.githubusercontent.com/9019452/48097075-dcec4700-e1e6-11e8-9d5b-eb27c41efd2d.png)

Reply:
Woo, looking nice! If you want, you can grab Forge-style pagination from Aurora [by copying over this template](https://github.com/DoSomething/aurora/blob/master/resources/views/vendor/pagination/default.blade.php) (which I believe Laravel should auto-register if put in the same path).